bin/compile.php

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env php
+<?php
+
+require __DIR__ . '/../src/FileSystemTraversal.php';
+$file_system_traversal = new \Gnorm\FileSystemTraversal();
+
+$repo_root = $file_system_traversal->findRepoRoot();
+$autoload = require_once $repo_root . '/vendor/autoload.php';
+
+if (!isset($autoload)) {
+    print "Unable to find autoloader for Gnorm.\n";
+    exit(1);
+}
+
+/**
+ * Options:
+ * c = The json encoded config.
+ * b = Boolean telling if this is a build.
+ */
+$options = getopt('c:b::');
```

Comment:
This optional "b" option does not appear to be working as expected.  Changing it to required (`b:` vs `b::`) appears to fix it.